safe/report/extractors/composer.py

```diff
@@ -222,12 +222,16 @@ def qgis_composer_extractor(impact_report, component_metadata):
         # place exposure at the bottom
         layers.append(impact_report.impact_function.exposure)
 
+    # default extent is impact extent
+    if not qgis_context.extent:
+        qgis_context.extent = impact_report.impact_function.analysis_extent
```

Comment:
Yes, default to analysis_extent if qgis_context.extent was not set before report generation.
So, this is one is the correct approach, right?